datahub-web-react/src/app/entity/tag/TagProfile.tsx

```diff
@@ -32,6 +48,29 @@ const TitleText = styled(Typography.Title)`
     }
 `;
 
+const HeaderLayout = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+
+const StatsBox = styled.div`
+    width: 180px;
+    justify-content: left;
+`;
+
+const StatText = styled(Typography.Text)`
+    font-size: 15px;
```

Comment:
Nit: Should we keep multiples of 4 on font sizes also?

Reply:
I don't think that necessary? I notice antd's "default" size is 14px which is off the 4px grid anyway. Their headers jump by units of 4px, but I think the spaces between the header sizes are probably too large for labels and layouts. Let's talk about this more though & come up with some standard font sizes to use